scripts/notebooks/SampleQC.ipynb

```diff
@@ -0,0 +1,2819 @@
+    "    plt.grid(True, zorder=0)\n",
+    "    \n",
+    "    # Adding a scatter plot to the figure with data points colored according to sex category\n",
+    "    plt.scatter(all_cn.chrX_CopyNumber, all_cn.chrY_CopyNumber, alpha=1, s=10,\n",
+    "               c=color_sex_list, zorder=2)\n",
+    "    \n",
+    "    # Setting the x and y axis limits\n",
+    "    plt.xlim([-0.1,3.15])\n",
+    "    plt.ylim([-0.1,3.1])\n",
+    "    \n",
+    "    # Setting the x and y axis ticks\n",
+    "    plt.xticks(cn_list)\n",
+    "    plt.yticks(cn_list)\n",
+    "\n",
+    "    # Adding text to the plot representing different combinations of X and Y chromosomes\n",
+    "    for nx in cn_list:\n",
+    "        for ny in cn_list:\n",
+    "            if nx + ny >=6:\n",
+    "                continue\n",
+    "            plt.text(nx, ny, nx*\"X\" + ny*\"Y\", color='lightgray', ha='center', va='center', zorder=1, weight=\"bold\")\n",
+    "    \n",
+    "    # Adding a legend to the plot\n",
+    "    plt.legend(handles=[Line2D(\n",
+    "        [0],[0], alpha=1, marker='o', color='w', \n",
+    "        markerfacecolor=color_dict[label], \n",
+    "        label=label + xy_rep[label]\n",
+    "    ) for label in color_dict.keys()], framealpha=1, labelspacing=0.5)\n",
+    "    \n",
+    "    # Adding a title and labels to the plot\n",
+    "    plt.title(f\"Sex Chromosome Ploidy - {len(sex_for_fig)} Samples\", fontsize=15)\n",
+    "    plt.xlabel(\"chrX Copy Number\", fontsize=13)\n",
+    "    plt.ylabel(\"chrY Copy Number\", fontsize=13)\n",
+    "        \n",
+    "    # Save the plot as an image    \n",
+    "    file_name = f\"sex_chromosome_ploidy_{len(sex_for_fig)}_samples.png\"\n",
+    "    file_path = generate_file_path(TLD_PATH, 'sex_analysis', file_name)\n",
+    "    save_figure(WS_BUCKET, file_path)\n",
+    "    \n",
+    "    # Show figure if display = True, else close to free up memory and return file name\n",
+    "    if (display == True):\n",
+    "        plt.show()\n",
+    "        return\n",
+    "    plt.close()\n",
+    "    return output_png"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Data Ingestion\n",
+    "This section fetches the sample QC data."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Step 1. Sample Sets\n",
+    "This step loads the sample_set data table to find the QC data file paths. You can exclude unnecessary sample_sets at this stage if needed."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class=\"alert alert-block alert-info\">Uncomment and run once. Once this step has run, if you need to load the table again, use the following cell.</div>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "code_folding": []
+   },
+   "outputs": [],
+   "source": [
+    "# sample_set_response = fapi.get_entities_tsv(\n",
```

Comment:
In Jupyter Notebooks, you can simply do `⌘` + `/` to comment/uncomment an entire code-block. I thought this would be easier that deleting/undeleting the `"""`. Happy to change this if you think it's more suitable though.